docarray/array/queryset/lookup.py

```diff
@@ -106,6 +106,8 @@ def lookup(key, val, doc: 'Document') -> bool:
         return iff_not_none(value, lambda y: re.search(val, y) is not None)
     elif last == 'size':
         return iff_not_none(value, lambda y: len(y) == val)
+    elif last == 'exists':
+        return (value is None or value == '' or value == b'') != val
```

Comment:
shouldn't `bool(value)` do the trick?

Reply:
in case `Document(tags={'x':0})`, then the query `{'tags_x': {'$exists': True}}` should return True